packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/views/details/services/FormDescriptionAggregator.java

```diff
@@ -10,7 +10,7 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
-package org.eclipse.sirius.components.collaborative.forms;
```

Comment:
Maybe rename to something that show this class is related to the details view only?